gnuradio-runtime/python/gnuradio/ctrlport/monitor.py

```diff
@@ -40,7 +40,7 @@ def __init__(self,tool="gr-ctrlport-monitor"):
                 gr.prefs().singleton().set_bool("ControlPort","edges_list",True);
                 gr.prefs().singleton().set_bool("PerfCounters","on",True);
                 gr.prefs().singleton().set_bool("PerfCounters","export",True);
-        except:
+        except RuntimeError:
             print("no support for gr.prefs setting")
```

Comment:
Looking at the perfs implementation, it is not clear to me how there would not be support for settings on these, but if that is the case, this exception will catch the non implemented functions.

Reply:
Removed this check so we get an error if that is broken for some reason.